conda_forge_admin_requests/mark_broken.py

```diff
@@ -89,33 +90,46 @@ def run(request):
     if did_any:
         with tempfile.TemporaryDirectory() as tmpdir:
             subprocess.check_call(
-                "git clone https://github.com/conda-forge/"
-                "conda-forge-repodata-patches-feedstock.git",
+                [
+                    "git",
+                    "clone", 
+                    "https://github.com/conda-forge/conda-forge-repodata-patches-feedstock.git",
+                ],
                 cwd=tmpdir,
-                shell=True,
             )
 
-            subprocess.check_call(
-                "git remote set-url --push origin "
+            origin_url = (
                 "https://x-access-token:${GITHUB_TOKEN}@github.com/conda-forge/"
```

Comment:
Not sure if `git` recognizes this env var syntax on its own or whether we were relying on shell expansion to populate this.

Reply:
We are relying on shell expansion here.